v2/pkg/engine/resolve/authorization_test.go

```diff
@@ -205,7 +205,7 @@ func TestAuthorization(t *testing.T) {
 		resolveCtx := &Context{ctx: context.Background(), Variables: nil, authorizer: authorizer}
 
 		return res, resolveCtx,
-			`{"errors":[{"message":"Unauthorized request to Subgraph 'users' at Path 'query', Reason: Not allowed to fetch from users Subgraph."}],"data":null}`,
+			`{"errors":[{"message":"Unauthorized request to Subgraph 'users' at Path 'query', Reason: Not allowed to fetch from users Subgraph."},{"message":"Failed to fetch from Subgraph 'reviews' at Path 'query.me'.","extensions":{"errors":[{"message":"could not render fetch input","path":["me"]}]}},{"message":"Failed to fetch from Subgraph 'products' at Path 'query.me.reviews.@.product'.","extensions":{"errors":[{"message":"could not render fetch input","path":["me","reviews","@","product"]}]}}],"data":null}`,
```

Comment:
Should we expose errors like this? Rendering the fetch input is an internal that I think should not be exposed.

Reply:
I think a better solution could be to determine what is the cause of error during the error, e.g. wrong input.
In general, such an error should almost never happen because we validate the input from the user.
So the only possibility is that we've got data back from a Subgraph that's invalid, so I think such a scenario is more like an internal error. We should log it, but not expose too many details to the client.

Reply:
Those errors have been propagated because I configured the `SelectResponseErrorsPath`. My PR did not touch this.

Reply:
Also don't forget those tests are running with `PropagateSubgraphStatusCodes` enabled.